add particle docs (#375)
moonbeam-foundation/moonbeam-docs-cn: builders/integrations/wallets/particle-network.md

---
title: Add Particle Network to a dApp - 将Particle Network集成至DApp
description: Learn how to integrate Particle Network's Wallet-as-a-Service into a dApp built on Moonbeam to enable MPC-based onboarding and ERC-4337 AA interaction.
学习如何将Particle Network的智能钱包即服务集成至Moonbeam上的dApp中，从而实现基于MPC入门和ERC-4337账户抽象交互。
---

# Particle Network Smart Wallet-as-a-Service - Particle Network的智能钱包即服务

## Introduction - 概览 {: #introduction }

[Particle Network](https://particle.network/){target=_blank} is a [Smart Wallet-as-a-Service](https://docs.particle.network/getting-started/smart-wallet-as-a-service){target=_blank} provider that allows developers to improve user experience through modular and customizable Externally Owned Account (EOA) and [Account Abstraction (AA)](https://docs.particle.network/developers/account-abstraction){target=_blank} embedded wallet components.

> [Particle Network](https://particle.network/){target=_blank}是一个[智能钱包即服务](https://docs.particle.network/getting-started/smart-wallet-as-a-service){target=_blank}提供商，允许开发者通过模块化和可定制化的外部拥有账户（Externally Owned Account，简称EOA）和[账户抽象（Account Abstraction，简称AA）](https://docs.particle.network/developers/account-abstraction){target=_blank}嵌入式钱包组件提升用户体验。

Reviewer: ```suggestion
[Particle Network](https://particle.network/){target=_blank}是一个[智能钱包即服务](https://docs.particle.network/getting-started/smart-wallet-as-a-service){target=_blank}提供商，它允许开发者通过使用可定制的模块化外部拥有账户（Externally Owned Account，简称EOA）和[账户抽象（Account Abstraction，简称AA）](https://docs.particle.network/developers/account-abstraction){target=_blank}嵌入式钱包组件来提升用户体验。
```

创建好应用程序并安装完依赖项后，您可以开始配置项目了。

To configure your project, you'll need to:

配置项目需要执行以下步骤：

1. Import [`ParticleNetwork`](https://docs.particle.network/developers/auth-service/sdks/web#step-2-setup-developer-api-key){target=_blank} from [`@particle-network/auth`](https://docs.particle.network/developers/auth-service/sdks/web){target=_blank}

    从 [`@particle-network/auth`](https://docs.particle.network/developers/auth-service/sdks/web){target=_blank}导入[`ParticleNetwork`](https://docs.particle.network/developers/auth-service/sdks/web#step-2-setup-developer-api-key){target=_blank}

2. Import `Moonbeam` from [`@particle-network/chains`](https://docs.particle.network/developers/other-services/node-service/evm-chains-api){target=_blank}, as you'll need Moonbeam's chain name and ID in the next step

    您将在下一个步骤用到Moonbeam的链名称和ID，因此您需要先从[`@particle-network/chains`](https://docs.particle.network/developers/other-services/node-service/evm-chains-api){target=_blank}导入`Moonbeam`

3. Initialize `ParticleNetwork` using your project credentials that you retrieved from your dashboard and Moonbeam's chain name and ID, and, optionally, you can configure wallet display and security settings

>     使用从数据面板检索的项目凭据以及Moonbeam的链名称和ID初始化`ParticleNetwork`，并且您可以选择配置钱包显示和安全设置

Reviewer: ```suggestion
    使用您的项目凭据以及Moonbeam的链名称和ID初始化`ParticleNetwork`，并且您可以在这里选择配置钱包显示和安全设置
```

If you want to use an ERC-4337 AA, you'll need to take a couple additional steps:

如果您想要使用ERC-4337账户抽象，您需要执行一些额外的步骤：

1. Import `SmartAccount` from [`@particle-network/aa`](https://docs.particle.network/developers/account-abstraction/sdks/web){target=_blank} and `ParticleProvider` from [`@particle-network/auth`](https://docs.particle.network/developers/auth-service/sdks/web){target=_blank}

   从[`@particle-network/aa`](https://docs.particle.network/developers/account-abstraction/sdks/web){target=_blank}和[`@particle-network/auth`](https://docs.particle.network/developers/auth-service/sdks/web){target=_blank}导入`SmartAccount`

2. Initialize a `ParticleProvider` to handle wallet connect requests using the `particle` instance you created in the previous set of steps

   使用您在上述步骤创建的`particle`实例，初始化`ParticleProvider`以处理钱包连接请求

3. Initialize a [`SmartAccount`](https://docs.particle.network/developers/account-abstraction/sdks/web#initialize-the-smartaccount){target=_blank} using the provider, your project credentials, and the AA implementation and configuration. For Moonbeam, you'll use the `simple` implementation and pass in Moonbeam's chain ID

>    使用提供商、项目凭证以及账户抽象实现和配置初始化[`SmartAccount`](https://docs.particle.network/developers/account-abstraction/sdks/web#initialize-the-smartaccount){target=_blank}。对于Moonbeam来说，您需要使用`simple`实现并传入Moonbeam链ID

Reviewer: ```suggestion
   使用提供商、项目凭证以及账户抽象实现和配置来初始化[`SmartAccount`](https://docs.particle.network/developers/account-abstraction/sdks/web#initialize-the-smartaccount){target=_blank}。对于Moonbeam来说，您需要使用`simple`实现并传入Moonbeam链ID
```
